lib/config/options/index.ts

```diff
@@ -68,6 +68,31 @@ const options: RenovateOptions[] = [
     default: false,
     globalOnly: true,
   },
+  {
+    name: 'mergeConfidenceEndpoint',
+    description:
+      'If set, Renovate will query this API for Merge Confidence data.',
+    type: 'string',
+    advancedUse: true,
+    globalOnly: true,
+  },
+  {
+    name: 'mergeConfidenceSupportedDatasources',
+    description:
+      'If set, Renovate will query the merge-confidence JSON API only for datasources that are part of this list.',
+    allowedValues: [
+      'go',
+      'npm',
+      'maven',
+      'pypi',
+      'nuget',
+      'packagist',
+      'rubygems',
+    ],
+    type: 'array',
+    subType: 'string',
+    globalOnly: true,
+  },
```

Comment:
We can have one since the fallback value is 
```
export const supportedDatasources = [
  'go',
  'maven',
  'npm',
  'nuget',
  'packagist',
  'pypi',
  'rubygems',
];
```

Reply:
I think we should have a default list and it's non-mergeable so users can make their own reduced list